tools/mayan-edms/.mayan-env

```diff
@@ -0,0 +1,64 @@
+##################################################################
```

Comment:
left this file mostly intact. We can change it more later and as it is is mostly examples of what could be set.